src/i18n.module.ts

```diff
@@ -152,7 +152,7 @@ export class I18nModule implements OnModuleInit, OnModuleDestroy, NestModule {
     consumer
       .apply(I18nMiddleware)
       .forRoutes(
-        isNestMiddleware(consumer) && usingFastify(consumer) ? '(.*)' : '*',
+        isNestMiddleware(consumer) && usingFastify(consumer) ? '(.*path)' : '*path',
```

Comment:
As both Fastify and Express now use the latest path-to-regexp package there's no need to have different formats.
```suggestion
        '*path'
```